src/main/java/raisetech/StudentManagement/exception/handler/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,41 @@
+package raisetech.studentmanagement.exception.handler;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import raisetech.studentmanagement.exception.StudentNotFoundException;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+  /**
+   * ControllerのgetStudentメソッドで存在しないIDがリクエストされた際の例外をハンドリングする
+   * @param ex StudentNotFoundException
+   * @return 受講生ID: "+ id + "が見つかりません。
+   */
+  @ExceptionHandler(StudentNotFoundException.class)
+  public ResponseEntity<String> handleStudentNotFoundException(StudentNotFoundException ex){
```

Comment:
APIなのでレスポンスは text でなく json がよいですね。
```suggestion
  public ResponseEntity<Map<String, String>> handleStudentNotFoundException(StudentNotFoundException ex){
```